arm-recoveryservices/2016-06-01/swagger/vaults.json

```diff
@@ -615,8 +684,7 @@
       ],
       "properties": {
         "properties": {
-          "$ref": "#/definitions/VaultProperties",
-          "x-ms-client-flatten": true
+          "$ref": "#/definitions/VaultProperties"

```

Comment:
There seems to be a bug in our RP at this point where this structure bodes well with the service. When I applied client flattening, there were issues during Create Vault API call.